CMakeLists.txt

```diff
@@ -192,6 +192,13 @@ cmake_dependent_option(
 
 set(ONNX_NAMESPACE "onnx_torch" CACHE STRING "A namespace for ONNX; needed to build with other frameworks that share ONNX.")
 
+# This is a fix for a rare build issue on Ubuntu:
+# symbol lookup error: miniconda3/envs/pytorch-py3.7/lib/libmkl_intel_lp64.so: undefined symbol: mkl_blas_dsyrk
+# https://software.intel.com/en-us/articles/symbol-lookup-error-when-linking-intel-mkl-with-gcc-on-ubuntu
+if(UNIX AND NOT APPLE)
```

Comment:
Is `NOT APPLE` necessary here? The same error may occur if some user on MacOS has customized default compiler configuration to something similar to the default of Ubuntu.

Reply:
It fails on pytorch_macos_10_13_cuda9_2_cudnn7_py3_build: ld: unknown option: --no-as-needed

Reply:
I see, makes sense, macOS's default linker does not support this option. You can go for `if(LINUX)` if it feels better, as I don't think other BSD systems (if we would ever support them)  ship a linker that supports this option.